chatwoot/chatwoot
# feat: Rewrite command bar mixin to a composable

# Pull Request Template

## Description

This PR replaces the usage of mixins with the composable for the CMD bar and updates the `commandbar.vue` component to the Composition API.

**Files updated**
1. dashboard/routes/dashboard/commands/conversationHotKeys.js
2. dashboard/routes/dashboard/commands/appearanceHotKeys.js 
3. dashboard/routes/dashboard/commands/bulkActionsHotKeys.js 
4. dashboard/routes/dashboard/commands/inboxHotKeys.js
5. dashboard/routes/dashboard/commands/goToCommandHotKeys.js 

Fixes https://linear.app/chatwoot/issue/CW-3458/rewrite-command-bar-mixin-to-a-composable

## Type of change


## How Has This Been Tested?

**Test case**

1. Test all conversation actions from both the dashboard and inbox views using the CMD bar.
6. Test snooze notifications from the inbox view using the CMD bar.
7. Test bulk actions using the CMD bar.
8. Change the appearance using the CMD bar.
9. Verify that all navigation functions correctly through the CMD bar.


## Checklist:
